std/complex.d

```diff
@@ -788,6 +791,7 @@ Complex!T sin(T)(Complex!T z)  @safe pure nothrow @nogc
 ///
 unittest
 {
+    static import std.math;
```

Comment:
Hm... didn't notice this.

Is the correct fix to re-space the import statement, or to fix the asserts (they are only indented 2 spaces)?